iampolicymanagementv1/iam_policy_management_v1.go

```diff
@@ -15,7 +15,7 @@
  */
 
 /*
- * IBM OpenAPI SDK Code Generator Version: 3.78.0-67aec9b7-20230818-174940
```

Comment:
Not a problem with this PR, but think about installing the latest generator version prior to your next PR :)

Reply:
Good catch, thank you. Will do!